vendor/DOtherSide/lib/src/Status/AppDelegate.mm

```diff
@@ -0,0 +1,39 @@
+#include "DOtherSide/Status/AppDelegate.h"
+
+#include <QString>
+#include <QUrl>
+
+#import <AppKit/NSApplication.h>
+
+
+@interface AppDelegate: NSObject <NSApplicationDelegate>
+- (BOOL)application:(NSApplication *)application
+continueUserActivity:(NSUserActivity *)userActivity
+ restorationHandler:(void (^)(NSArray<id<NSUserActivityRestoring>> *restorableObjects))restorationHandler;
+@end
+
+@implementation AppDelegate
+- (BOOL)application:(NSApplication *)application continueUserActivity:(NSUserActivity *)userActivity restorationHandler:(void (^)(NSArray<id<NSUserActivityRestoring>> *))restorationHandler {
+    if (userActivity.activityType == NSUserActivityTypeBrowsingWeb) {
+        NSURL *url = userActivity.webpageURL;
+        if (!url) {
+           return FALSE;
+        }
+        QUrl deeplink = QUrl::fromNSURL(url);
+        // set it to nim
```

Comment:
Is this going to happen in future prs, or something is lost?

Reply:
yes :)